go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2236,10 +2236,11 @@ func (e *Executor) updateMigrationProgress(ctx context.Context, uuid string, pro
 	return err
 }
 
-func (e *Executor) retryMigration(ctx context.Context, whereExpr string) (result *sqltypes.Result, err error) {
+// retryMigrationWhere retries a migration based on a given WHERE clause
+func (e *Executor) retryMigrationWhere(ctx context.Context, whereExpr string) (result *sqltypes.Result, err error) {
 	e.migrationMutex.Lock()
 	defer e.migrationMutex.Unlock()
-	parsed := sqlparser.BuildParsedQuery(sqlRetryMigration, ":tablet", whereExpr)
+	parsed := sqlparser.BuildParsedQuery(sqlRetryMigrationWhere, ":tablet", whereExpr)
```

Comment:
just noticed that three lines down, on line 2247, we are never checking err from `parsed.GenerateQuery`

Reply:
Fixed!